src/components/Todo.vue

```diff
@@ -1,44 +1,54 @@
 <template>
   <div v-bind:class="{ 'completed': todo.completed, 'important': todo.important }">
     <p>
-        <input type='checkbox' v-on:change='markComplete' class='checkbox' />
-        <span class='todo-title' contenteditable="true" v-on:keydown.enter="updateTask($event, list)" v-on:blur="updateTask($event, list)">{{ todo.title }}</span>
-        <button class='btn btn-outline-dark' v-on:click="markImportant">!</button>
-        <button class='btn btn-outline-danger' @click="$emit('delete-todo', todo.id)">X</button>
+      <input type="checkbox" v-on:change="toggleComplete" class="checkbox" />
+      <span
+        class="todo-title"
+        contenteditable="true"
+        v-on:blur="updateTitle($event)"
+      >{{ todo.title }}</span>
+      <button class="btn btn-outline-dark" v-on:click="toggleImportant">!</button>
+      <button class="btn btn-outline-danger" @click="$emit('delete-todo', todo.id)">X</button>
     </p>
-    
   </div>
 </template>
 <script>
 export default {
-  name: 'Todo',
-  props: [
-    "todo"
-  ],
+  name: "Todo",
+  props: ["todo"],
   methods: {
-    markComplete() {
-      this.todo.completed = !this.todo.completed
+    toggleComplete() {
```

Comment:
method is called toggle and it's doing couple of things. Maybe to rename it or divide logic into separate methods